src/inim.nim

```diff
@@ -2,8 +2,14 @@
 # Copyright (c) 2018 Andrei Regiani
 
 import os, osproc, strformat, strutils, terminal,
-       times, strformat, parsecfg
+       times, strformat, parsecfg, algorithm
 import noise
+import macros
```

Comment:
you love to see it

Reply:
Haha, I now realize I can move this to commands